analysis/import/SCMLogParser/src/main/java/de/maibornwolff/codecharta/model/input/metrics/CodeChurn.java

```diff
@@ -0,0 +1,26 @@
+package de.maibornwolff.codecharta.model.input.metrics;
+
+import de.maibornwolff.codecharta.model.input.Modification;
+
+public final class CodeChurn implements ModificationMetric<Integer> {
+    public static final String NUMBER_OF_COMMITS = "code_churn";
```

Comment:
NUMBER_OF_COMMITS should be named CODE_CHURN, or since it is just a string for the metric name, METRIC_NAME. It also is not used anywhere else and can therefore be private, as far as i can see.
Since metricName() is only used to get a specific value of the instance, it is OK in my opinion to return the string instead of the static final constant. The meaningless string value gets its context and meaning from the method name metricName() and a named constant is not needed in my opinion.
There are some other metric classes which have the same naming/visibility issues. 
Apart from these minor things, I like the concept behind the model implementation. Easy to understand and use.